a2-synchrotron-contiki/core/net/mac/chaos/chaos.c

```diff
@@ -860,7 +869,15 @@ chaos_round(const uint16_t round_number, const uint8_t app_id, const uint8_t* co
     /* move to next slot */
     slot_number++;
     /* change channel */
+  #if CHAOS_CLUSTER
+    if(IS_CLUSTER_HEAD() && IS_CLUSTER_ROUND() && HAS_CLUSTER_ID()) {
```

Comment:
create trello card to remember IS_CLUSTER_ROUND is always 0 and HOP_CHANNEL_CLUSTER_HEAD is never used?

Reply:
Added